tensorflow_probability/python/mcmc/sample.py

```diff
@@ -100,10 +100,13 @@ def sample_chain(
   instance (`kernel`).
 
   This function can sample from multiple chains, in parallel. (Whether or not
-  there are multiple chains is dictated by the `kernel`.)
+  there are multiple chains is dictated by the `kernel`).
 
   The `current_state` can be represented as a single `Tensor` or a `list` of
-  `Tensors` which collectively represent the current state.
+  `Tensors` which collectively represent the current state. Note, if one desires
+  to sample from multiple chains in parallel, the `current_state` must be a
+  `Tensor` with a leading batch dimension representing the number of parallel
+  chains.
```

Comment:
The chain shape for parallel sampling is determined by the shape of `target_log_prob(*state)`. So the overall assertion that sample_chain can't do parallel sampling of multiple chains with a list of states is not correct.

Reply:
Ah got it. Just for my own understanding, for a list of `Tensor`s to work, that means the `target_log_prob` would've had to be defined to accept an arbitrary number of args, right?

I guess I'm wondering now if you think a more accurate Docstring edit could be useful. At least personally, it took some time for me to debug why parallel sampling wasn't working.

Reply:
To be clear, @brianwa84 , @rupei is trying to say that to run a batch of chains, the batching has to be across a Tensor dimension, not across a list.  List structure is always interpreted as part of the latent state of one chain, never as indexing a batch dimension.